CHANGELOG.md

```
@@ -1,5 +1,20 @@
 # Changelog
 
+## [v0.2.0] - 2026-04-05
+
+### Added
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)